StrongInject.Generator/InstanceSource.cs

```diff
@@ -118,6 +118,18 @@ public static InstanceSource Create(INamedTypeSymbol asType, InstanceSource unde
                 ? underlying
                 : new ForwardedInstanceSource(asType, underlying is ForwardedInstanceSource forwardedUnderlying ? forwardedUnderlying.Underlying : underlying);
 
+        public override void Visit<TState>(IVisitor<TState> visitor, TState state)
+        {
+            visitor.Visit(this, state);
+        }
+    }
+    internal record OwnedSource(
+        ITypeSymbol ContainerType,
```

Comment:
Can we call this OwnedType? `Container` is quite overloaded in this context.

Reply:
How about `OwnerType` since the type is either `Owner<T>` or `AsyncOwner<T>`? `OwnedType` makes me think it means the `T` type.

Reply:
Well the name is `Owned<T>`. Maybe `OwnedWrapperType`? I don't really have a good name for this.

Reply:
Oh ha, wonder how I made that mistake. I guess `OwnedType` next to `ValueType` is less ambiguous. Maybe it should be `OwnedType`+`OwnedValueType`.

Reply:
👍